theano/sandbox/gpuarray/elemwise.py

```diff
@@ -69,9 +86,11 @@ def __str__(self):
 
     def make_node(self, *inputs):
         res = Elemwise.make_node(self, *inputs)
+        inputs = [as_gpuarray_variable(i, context=self.context)
```

Comment:
Why put a context in the op itself for each op with multiple input? I think I would just call infer_context(*inputs) and use that contect here.